interface/skin.dmf

```diff
@@ -147,6 +147,7 @@ window "mapwindow"
 		is-default = true
 		saved-params = "icon-size"
 		zoom-mode = distort
+		style = "body { font-family: 'Verdana'; font-size: 7px; }\n.vga { font-family: 'PxPlus IBM VGA9'; font-size: 12px; } .ps2p { font-family: 'Press Start 2P'; font-size: 6px; }\n.pixel { font-family: 'Small Fonts'; font-size: 7px; }\n.ol{ -dm-text-outline: 1px black; }\n.sh { text-shadow: 1px 1px 0 black; }\n.l { text-align: left; }\n.c { text-align: center; }\n.r { text-align: right; }\n.vt { vertical-align: top; }\n.vm { vertical-align: middle; }\n.vb { vertical-align: bottom; }\n.reliq { font-family: 'Inhuman BB'; font-size: 12px; }"

```

Comment:
Woah! Would you look at this! Same exact line as 
https://github.com/goonstation/goonstation/blob/8daee35ab6bbb9a058cc192e290aa12b670f2dfe/interface/skin.dmf#L795

Gee, what a coincidence we ended up with the same exact CSS. You guys don't even implement the majority of this CSS, or our various stylings. I guess something that looks like our code exactly just landed wholesale in your lap. Or you stole and illegally re-licensed it, violating our code license.